pt/sessions/codewars.html

```diff
@@ -1,51 +1,50 @@
-<h4>Code Wars Beginner Kata</h4>
-<p>I'd recommend doing the basics first, but then go back and forth between
+----------
+### Code Wars Beginner Kata
+----------
+I'd recommend doing the basics first, but then go back and forth between numbers and strings as they get more difficult. The later ones have more complex loops, it might be better to do all the simpler first. Last move to the group I've labeled advanced. Some of this are kinda challenging with the concepts we've covered, some just didn't fit well in other sections.
 
-numbers and strings as they get more difficult. The later ones have more
+----------
+#### Basics and Bools - do these first
 
-complex loops, it might be better to do all the simpler first. Last move
-
-to the group I've labeled advanced. Some of this are kinda challenging
-
-with the concepts we've covered, some just didn't fit well in other sections.
-</p>
-<h4>Basics and Bools - so these first</h4>
 <ul>
-<li><a href=">https://www.codewars.com/kata/50ee6b0bdeab583673000025">basic variable assignment</a></li>
-<li><a href="https://www.codewars.com/kata/57cc975ed542d3148f00015b">You only need one</a></li>
-<li><a href="https://www.codewars.com/kata/568dcc3c7f12767a62000038">set alarm</a></li>
+<li><a target="_blank" href="https://www.codewars.com/kata/50ee6b0bdeab583673000025">Basic variable assignment</a></li>
```

Comment:
Can _blank be achieved with markdown? Would prefer not to have to use html here.